modules/odc-using-web-terminal.adoc

```diff
@@ -18,7 +18,7 @@ After the Web Terminal Operator is installed, you can use the web terminal as fo
 +
 . Click *Start* to initialize the web terminal using the selected project.
 +
-After the web terminal is initialized, you can use the preinstalled CLI tools like `oc`, `kubectl`, `odo`, `kn`, `tkn`, `helm`, `kubens`, `subctl`, and `kubectx` in the web terminal.
```

Comment:
Are we completely removing any reference to `odo`? `odo` will still be a preinstalled tool when the web terminal is initialized, shouldn't it still be here?

Reply:
Thanks for your feedback. Revoking the change here.